components/dashboard/Task.js

```diff
@@ -114,19 +117,30 @@ export default function Task(props) {
   }
 
   return (
-    <Card className={clsx(classes.card, highlightLabel && classes.highlight)} data-intercom="task">
+    <Card
+      className={clsx(classes.card, highlightLabel && classes.highlight)}
+      data-intercom="task"
+      elevation={0}
+    >
       {!!highlightLabel && <div className={classes.highlightLabel}>{highlightLabel}</div>}
-
-      <CardHeader
-        title={<strong>{task.fields.Title}</strong>}
-        titleTypographyProps={{ component: 'h1', variant: 'h3' }}
-      />
+      <CardHeader title={task.fields.Title} titleTypographyProps={{ component: 'h5' }} />
       <CardContent>
         {/* eslint-disable-next-line jsx-a11y/accessible-emoji */}
-        <div className={classes.timeEstimate} data-intercom="task-time-estimate">
-          🕒{task.fields['Time Estimate']} min.
+        <div className={classes.timeEstimate}>
+          <Typography variant="overline" component="p" style={{ lineHeight: '1.4' }}>
+            Estimated Time
+          </Typography>
+          <Typography
+            data-intercom="task-time-estimate"
```

Comment:
Let's keep the `data-intercom` attribute on the parent `div`, so that when the Intercom product tour highlights it, it highlights both the caption and the value.